src/Instrumentation/Psr18/README.md

```diff
@@ -0,0 +1,27 @@
+# OpenTelemetry PSR-18 auto-instrumentation
+
+## Requirements
+
+* OpenTelemetry extension
+* OpenTelemetry SDK and exporters (required to actually export traces)
+* A psr-18 client
+
+## Overview
+Auto-instrumentation hooks are registered via composer, and spans will automatically be created for each PSR-15 middleware that is executed.
+
+To export spans, you will need to create and register a `TracerProvider` early in your application's lifecycle:
+
+```php
+<?php
+require_once 'vendor/autoload.php';
+
+$tracerProvider = /*create tracer provider*/;
```

Comment:
Do we still need do this that way as https://github.com/open-telemetry/opentelemetry-php/pull/854 is already merged?

Reply:
I've updated the text so that sdk autoloading is preferred... how does it sound now?

Reply:
Sounds good to me